packages/snjs/lib/application.ts

```diff
@@ -864,21 +864,50 @@ export class SNApplication {
   }
 
   /**
-   * Returns all parents for a tag
+   * Establishes a hierarchical relationship between two tags.
+   */
+   public async setTagRelationship(
+    parentTag: SNTag,
+    childTag: SNTag
+  ): Promise<void> {
+    await this.itemManager.setTagRelationship(parentTag, childTag);
+  }
+
+  /**
+   * Remove the tag parent.
+   */
+  public async unsetTagRelationship(
+    parentTag: SNTag,
+    childTag: SNTag
+  ): Promise<void> {
+    await this.itemManager.unsetTagRelationship(parentTag, childTag);
+  }
```

Comment:
@moughxyz I wanted to add the "unset" operation, but I couldn't find a good antonym for `establishRelationship`, so I used `set` and `unset`, let me know if that works for you.

Note we could ignore `parentTag` in there, because in the current implementation we don't use it, I kept it anyway because:
- we can make the API symmetric with the set operation,
- we keep the option to change our internal representation later without breaking client code.